packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -98,7 +104,7 @@ const isSteve = () =>
 function wanderTask(
   details: Delayed<WanderDetails>,
   spec: Delayed<OutfitSpec>,
-  base: Omit<GarboTask, "outfit" | "do" | "choices" | "spendsTurn"> & {
+  base: Omit<GarboTask, "outfit" | "do" | "spendsTurn" | "choices"> & {
```

Comment:
why

Reply:
Was trying to do a thing, fucked up reverting them. Lemme fix that.